src/Api/MonaiApplicationEntity.cs

```diff
@@ -66,9 +69,27 @@ public class MonaiApplicationEntity
         public string AeTitle { get; set; }
 
         /// <summary>
-        /// Optional field to the AE to one or more applications.
+        /// Gets or sets the DICOM tag used to group the instances.
+        /// Defaults to 0020,000D (Study Instance UID).
         /// </summary>
-        public List<string> Applications { get; set; }
+        public string Grouping { get; set; } = "0020,000D";
+
+        /// <summary>
+        /// Optional field to map AE to one or more workflows.
+        /// </summary>
+        public List<string> Workflows { get; set; }
+
+        /// <summary>
+        /// Optional field to specify SOP Class UIDs to ignore.
+        /// </summary>
+        public List<string> IgnoredSopClasses { get; set; }
```

Comment:
This list will be long if making it exhaustive.

Reply:
New issue created to use allow-list. Although we have never seen anyone used this feature.

Reply:
It was because the responsibility to select the study/series was pushed to the DICOM sender AE in the upstream, and the Application's series selector in the down stream. Having a incomplete ignored list here does not help much, unless it is to fend off some edge case bad sender(?).